dockerfiles/che/entrypoint.sh

```diff
@@ -83,8 +83,10 @@ set_environment_variables () {
   ### Set value of derived environment variables.
 
   # CHE_DOCKER_IP is used internally by Che to set its IP address
-  if [[ -n "${CHE_IP}" ]]; then
-    export CHE_DOCKER_IP="${CHE_IP}"
+  if [ -z "${CHE_DOCKER_IP}" ];then
```

Comment:
Amazing answer https://stackoverflow.com/questions/3869072/test-for-non-zero-length-string-in-bash-n-var-or-var/3870055#3870055